packages/schema-blocks/src/blocks/BlockSuggestions.tsx

```diff
@@ -35,7 +35,7 @@ function BlockSuggestion( { blockTitle, blockName, blockClientId }: BlockSuggest
 	};
 
 	return (
-		<li className="yoast-block-suggestion">
+		<li className="yoast-block-suggestion" key={ blockTitle }>
```

Comment:
we want to be 100% unique, right?
```suggestion
		<li className="yoast-block-suggestion" key={ "BlockSuggestion" + blockTitle }>
```

Reply:
[Keys hoeven niet globaal uniek te zijn](https://reactjs.org/docs/reconciliation.html#recursing-on-children):
> The key only has to be unique among its siblings, not globally unique.

Ik mis een whiteboard om het uit te leggen 😅, maar [de React documentatie legt het goed uit](https://reactjs.org/docs/reconciliation.html#recursing-on-children).